assets/training/model_management/environments/foundation-model-inference/context/score.py

```diff
@@ -276,28 +233,25 @@ async def async_analyze_text_task(client, request):
 
 
 def analyze_response(response):
-    """Analyze response."""
     severity = 0
 
     if response.hate_result is not None:
-        logger.info("Hate severity: {}".format(response.hate_result.severity))
+        print("Hate severity: {}".format(response.hate_result.severity))
         severity = max(severity, response.hate_result.severity)
     if response.self_harm_result is not None:
-        logger.info("SelfHarm severity: {}".format(response.self_harm_result.severity))
+        print("SelfHarm severity: {}".format(response.self_harm_result.severity))
         severity = max(severity, response.self_harm_result.severity)
     if response.sexual_result is not None:
-        logger.info("Sexual severity: {}".format(response.sexual_result.severity))
+        print("Sexual severity: {}".format(response.sexual_result.severity))
         severity = max(severity, response.sexual_result.severity)
     if response.violence_result is not None:
-        logger.info("Violence severity: {}".format(response.violence_result.severity))
+        print("Violence severity: {}".format(response.violence_result.severity))
         severity = max(severity, response.violence_result.severity)
 
     return severity
 
 
 def analyze_text_async(text):
```

Comment:
we need to add "global aacs_client" here, otherwise it was erroring out on line 262 as Nonetype

here and under def analyze_text as well

Reply:
Thanks for catching that!